frontend/src/app/components/header/header.component.html

```diff
@@ -5,42 +5,60 @@
   </button>
   <div class="collapse navbar-collapse" id="navbarCollapse">
     <!-- Left Grouping -->
-    <ol class="navbar-nav mr-auto">
-      <li>
-        <a class="navbar-link" routerLink="">
-          <img style="height: 3em ;" src="assets/img/standapp_logo.png">
-        </a>
+    <li>
+      <a class="navbar-link" routerLink="">
+        <img style="height: 3em ;" src="assets/img/standapp_logo.png">
+      </a>
+    </li>
+    <ol class="breadcrumb" *ngIf="routerUrl.url == '/teams' || routerUrl.url == '/' && loggedIn" >
+      <li class="breadcrumb-item active" aria-current="page">
+        <a routerLink="teams">My Teams</a>
       </li>
-      <li class="nav-item">
-        <a class="nav-link" routerLink="about">About Us</a>
+    </ol>
+
+    <ol class="breadcrumb" *ngIf="routerUrl.url == '/team/11'">
```

Comment:
based on your PR comment... I'm guessing that it the 'equals' string comparison from the == in the *ngIf statement. Can check for its existence using just `*ngIf="routerUrl.url` or if its reliant on the /team/ or /standup/ aspects (but not the id) then should be able to do some kind of string contains comparison (I don't immeadiately know the syntax for this), something like `*ngIf="routerUrl.url.contains('/team/')"`

Reply:
Kind of...
The issue isn't necessarily the comparison although it sounds like there is a more effective way to do it. It's getting the teamId and standupId from the other components without injecting it the way we did (or maybe I'm not doing it right). 

This is because to inject the way we have we need the <app-team> tags which screw up the view within the html for this. 

I think it'll make more sense if I show you next standup.